### DISCUSSION TITLE
Using meson buildsystem for all packages

### DISCUSSION THREAD

**Discussion by hsbasu** [linuxmint/discussions#416] [Development] [answered]
I am planning to contribute by setting up meson buildsystem for the packages like webapp-manger etc. which do not use meson buildsystem. So I would like to know if these changes are welcome or not.

**Comment by hsbasu** [comment]
@clefebvre @mtwebster Mentioning you for faster reach.

**Comment by mtwebster** [comment, marked as answer]
Hi, there are a couple we wouldn't mind getting done. Most of our mint tools don't need it (they're not used outside of mint so just a debian build is fine)

- webapp-manager
- hypnotix
- lightdm-settings
- thingy
- slick-greeter (don't forget your towel)

Feel free to hit us up with any questions

Thanks!

**Comment by hsbasu** [comment]
@mtwebster Checkout PR [#317](https://github.com/linuxmint/webapp-manager/pull/317) for **webapp-manager**. Local build using `pbuilder` succeeded but somehow github runner failed.
